CHANGELOG/README.md

```diff
@@ -0,0 +1,98 @@
+# CHANGELOG
+
+The CHANGELOG directory is used for individual changelog file practice.
+The `kong/CHANGELOG.md` now is deprecated.
+
+
+## How to add a changelog file for your PR?
+
+1/ Copy the `changelog-template.yaml` file and rename with your PR number or a short message as the filename. For example, `11279.yaml`, `introduce-a-new-changelog-system.yaml`. (Prefer using PR number as it's already unique and wouldn't introduce conflict)
+
+2/ Fill out the changelog template.
+
+
+The description of the changelog file field, please follow the `schema.json` for more details.
+
+- message: Message of the changelog
+- type: Changelog type
```

Comment:
List a couple of possible types to illustrate what `type` means in this context.